File: ProgramaAssistenciaTecnicaPython/NotaFiscal.py
class NotaFiscal:
    def __init__(self, valorManutencao):
        self.valorManutencao = valorManutencao

    def exibeNota(self):
        print("+", "-" *15, "+")
        print("| NOTA FISCAL |")
        print("|           PAGO|")
        print("+", "-" * 15, "+")

    def verificaPagamento(self, valorPago):
        if bool(valorPago):
            print("\n Situação: Pago")
        else:
            print("\n Situação: não pago")

File: aula02/pw-my-assist/src/NotaFiscal.java
public class NotaFiscal {
    private double valorManutencao;

    public NotaFiscal(double valorManutencao) {
        this.valorManutencao = valorManutencao;
    }

    public double getValorManutencao() {
        return valorManutencao;
    }

    public void setValorManutencao(double valorManutencao) {
        this.valorManutencao = valorManutencao;
    }

    public void exibeNota() {
        System.out.println("+---------------+");
        System.out.println("| NOTA FISCAL   |");
        System.out.println("|      PAGO     |");
        System.out.println("+---------------+");
    }

    public void verificaPagamento(boolean valorPago) {
        if (valorPago) {
            System.out.println("\n Situação: Pago");
        } else {
            System.out.println("\n Situação: não pago");
        }
    }
}
